DBTableToHBaseTable/README.rst

```diff
@@ -0,0 +1,85 @@
+=============================================================
+Batch Database Table To CDAP HBase Table Adapter Configuration
+==============================================================
+
+The ETL Batch Template can be used to create an Adapter that reads from a Batch Source and persists it to a Sink.
+In this example, we will read an entire DB table in batch and use a TableSink to write the database table's rows to HBase.
+
+The config.json contains a sample Adapter configuration that you can use to accomplish the above task. Our sample Adapter uses these components:
+
+- ETLBatch Application Template, since we want to perform ETL in batch
+- Database source, to read data from the database table 
+- Table sink, to write the Kafka events to a Stream
+- A jar file containing the JDBC driver for your database. Along with this, you also need a JSON file that describes the JDBC driver as an external plugin. This file should have the same name as the jar file (with only the extension changed to '.json'). See `mysql-connector-java-5.1.35.json`` and ``postgresql-9.4.json`` as examples.
+
+You can create and start the Adapter by using the CDAP CLI (or you can use the UI for a more visual approach).
+
+Note: You need to fill in the following configurations in a file such as config.json before creating the Adapter.
+
+Configurations for the Database Table Source
+--------------------------------------------
+
+  #. ``driverClass`` - This is the JDBC driver class used to connect to your database.
```

Comment:
I would start all these lines flush left, and then hang the bullets. Example:
```
#. ``user`` - This is the username used to connect to the specified database. It is required 
   for databases that need authentication, optional for those that do not.
```
The first character of the second and succeeding lines is lined up with the first character after the space in the first line.